pyaedt/modules/Boundary.py

```diff
@@ -296,9 +346,9 @@ class BoundaryObject(BoundaryCommon, object):
         An AEDT application from ``pyaedt.application``.
```

Comment:
The Google developer documentation guidelines say that a literal string should be used as an adjective, with a noun afterwards. Is ``pyaedt.application`` a module, a class, or what? Also, we generally omit the "A, An, or The" in the first summary line of a description.

Reply:
@PipKat It is a class, how should I modify it? 

For example?

app : :class:`pyaedt.application`
        Pyaedt object from ``pyaedt.application``.